requirements/framework.txt

```diff
@@ -1,4 +1,4 @@
-aiohttp==3.8.6
+aiohttp==3.9.0
```

Comment:
Consider upgrading to the latest 3.9.1: https://pypi.org/project/aiohttp/#history

Reply:
Good call! Done :)